### PR TITLE
Diff show empty lines

### PR DESCRIPTION
**Pull request by 0xDCA**
This PR modifies the CSS styles for diff lines so empty diff lines are always shown. This helps the user to debug differences of only an End Of Line

### CODE REVIEW

File: inginious/frontend/common/static/js/task.js [hunk 1]
```diff
@@ -537,10 +537,12 @@ function parseOutputDiff(diff) {
       output = '<span class="diff-missing-output">' + line.substring(1) + '</span>';
     } else if (line.startsWith("+")) {
       output = '<span class="diff-additional-output">' + line.substring(1) + '</span>';
-    } else if (line.startsWith(" ") || line === "") {
+    } else if (line.startsWith(" ")) {
       output = '<span class="diff-common">' + line.substring(1) + '</span>';
     } else if (line.startsWith("...")) {
       output = '<span class="diff-position-control">' + line + '</span>';
+    } else if (line === "") {
```
Comment (amrondonp) [inline inginious/frontend/common/static/js/task.js:544]: This isn't doing anything right?
Is like having this case si that it does not cause an execption?
I cannot fully understand this piece. I think a commentary saying why would be very helpful

**Review by amrondonp — COMMENTED** [review]
Great, just a question

File: inginious/frontend/common/static/js/task.js (hunk 1 above)
Comment (0xDCA) [reply inginious/frontend/common/static/js/task.js:544]: I added a comment

Comment (amrondonp) [reply inginious/frontend/common/static/js/task.js:544]: A lot clearer, Thanks

**Review by amrondonp — APPROVED** [review]
LGTM

**Pull request merged**